ci/run_tests_test.go

```diff
@@ -86,7 +86,7 @@ func TestRunTests(t *testing.T) {
 	ctx, cancel := runData.Assignment.WithTimeout(2 * time.Minute)
 	defer cancel()
 
-	scm, _ := qtest.FakeProviderMap(t)
+	scm, _ := qtest.TestSCMManager(t)
```

Comment:
Since this is in the `qtest` package, I think it would make sense to just name it `SCMManager`. On the other hand, I see that we often use only the `scm` value and not the manager itself. Perhaps we should have separate test helpers for these two ... but didn't check what is needed in current tests.

Reply:
I thought about splitting, but we need only the scm twice across all tests, both the scm and the manager twice, and only the manager four times, so I'm still not sure if it makes sense.